packages/angular/src/generators/setup-ssr/files/standalone/__serverFileName__

```diff
@@ -0,0 +1,60 @@
+import 'zone.js/dist/zone-node';
+
+import {APP_BASE_HREF} from '@angular/common';
+import {ngExpressEngine} from '@nguniversal/express-engine';
+import * as express from 'express';
+import {existsSync} from 'fs';
+import {join} from 'path';
+
+import bootstrap from './src/<%= main.slice(0, -3) %>';
+
+// The Express app is exported so that it can be used by serverless Functions.
+export function app(): express.Express {
+  const server = express();
+  const distFolder = join(process.cwd(), 'dist/apps/<%= project %>/browser');
```

Comment:
Should we rely on the project root for the directory? Normally that's what's used for the output folder. Something like:
```suggestion
  const distFolder = join(process.cwd(), 'dist/<%= projectRoot %>/browser');
```

Reply:
I'd say we could make this improvement later, this is the current behaviour of our SSR generator. I'm reluctant to change it as part of this support

Reply:
I do think you're right though, as it doesn't cover the standalone project structure either. But again, it's something we can improve later